tests/rest/admin/test_user.py

```diff
@@ -1488,7 +1488,7 @@ def test_create_user_mau_limit_reached_active_admin(self) -> None:
 
         if channel.code != HTTPStatus.OK:
             raise HttpResponseException(
-                channel.code, channel.result["reason"], channel.json_body
+                channel.code, channel.result["reason"], channel.result["body"]
```

Comment:
Why are we changing this?

Reply:
Good question. I think I just picked this up during a find-and-replace.

Reply:
Ahh, withtout this change:

```
tests/rest/admin/test_user.py:1491: error: Argument 3 to "HttpResponseException" has incompatible type "Dict[str, Any]"; expected "bytes"  [arg-type]
```